triaging.rst

```diff
@@ -94,7 +104,8 @@ invalid
     Used manually for PRs that do not meet basic requirements and
     automatically added by bedevere when PR authors attempt to merge maintenace
     branches into the master branch. During competitions, this label causes the
-    PR to not count towards the author's contributions.
+    PR to not count towards the author's contributions. This label is generally
```

Comment:
I'm not clear why the competitions are mentioned. I would delete that sentence since GitHub will recognize the contribution if accepted.

Reply:
> I'm not clear why the competitions are mentioned. I would delete that sentence since GitHub will recognize the contribution if accepted.

I'll wait on your response to my recent comment before applying this suggestion. I'm okay with removing this part though.

Reply:
Perhaps explicitly reword to: During events such as the October Hacktoberfest, this label will prevent the PR from counting toward the author's contributions.